package.json

```diff
@@ -32,8 +32,8 @@
   "homepage": "https://github.com/davidtheclark/react-aria-modal#readme",
   "dependencies": {
     "focus-trap-react": "^3.0.3",
-    "no-scroll": "^2.0.0",
-    "react-displace": "^2.1.4"
+    "no-scroll": "^2.1.0",
+    "react-displace": "^2.2.1"
```

Comment:
I'm not sure about those. I didn't manually upgrade them, but maybe upgrading react/react-dom causes this?